buildSrc/private/src/main/kotlin/androidx/build/clang/NativeTargetCompilation.kt

```diff
@@ -137,4 +140,29 @@ class NativeTargetCompilation internal constructor(
             }
         }
     }
+
+    /**
+     * JDK ships with JNI headers only for the current platform. As a result,we don't have access
+     * to cross-platform jni headers. They are mostly the same and we don't ship cross compiled code
+     * from GitHub so it is acceptable to use local JNI headers for cross platform compilation on
+     * GitHub.
+     */
+    private fun findJniHeadersInPlayground(
+        javaHome: File
+    ): List<File> {
+        var include = File(javaHome, "include")
+        if (!include.exists()) {
```

Comment:
why is this needed? Please leave a comment

Reply:
this is from my kotlin-sqlite-bindings copy :). i don't remember why so let me try removing it.